### DISCUSSION TITLE
Linux Mint 21.2 Cinnamon - Transparency Issue

### DISCUSSION THREAD

**Discussion by dautry** [lacymorrow/crossover#319] [Q&A]
Hi All,

I'm new to Linux and so far really impressed with Mint.  I read in the bug section a transparency issue and necessity to download/install a compositor.  I installed picom and compton but when I try to start them the system states that another compositor is running?  I searched windows/desktop settings and could not find any instructions on how to make adjustments.

Any help would be appreciated.  The app runs fine except when I lock the crosshair into place the background does not go away -- the transparency aspect is simply not working.

**Comment by lacymorrow** [comment]
Tons of folks with the same problem: #318 , #302
